core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -58,14 +59,16 @@ export class NavigationComponent {
 
   // whether user dashboard is enabled and accessible from the workspace
   public userSystemEnabled: boolean = environment.userSystemEnabled;
+  // flag to display a particular version in the current canvas
+  public displayParticularWorkflowVersion: boolean = false;
   public onClickRunHandler: () => void;
 
   // whether the disable operator button should be enabled
   public isDisableOperatorClickable: boolean = false;
   public isDisableOperator: boolean = true;
 
   public operatorCacheEnabled: boolean = environment.operatorCacheEnabled;
-  public isCacheOperatorClickable: boolean = false;
+  public isCacheOperatorClickable: boolean = true;
```

Comment:
is this change intentional?

Reply:
oops sorry about that, fixed in a new commit